lib/hammer_cli/output/fields.rb

```diff
@@ -4,14 +4,16 @@ module Fields
 
   class Field
 
-    attr_reader :label
-    attr_reader :path
+    attr_accessor :label
+    attr_accessor :path
+    attr_accessor :id
 
     def initialize(options={})
       @hide_blank = options[:hide_blank].nil? ? false : options[:hide_blank]
       @hide_missing = options[:hide_missing].nil? ? true : options[:hide_missing]
       @path = options[:path] || []
       @label = options[:label]
+      @id = options[:field_id] || label || options[:key]
```

Comment:
Is there a reason the option is called `:field_id` and the attribute just `id`? It is not consistent with other options.